src/Models/ApplicantModel.php

```diff
@@ -137,25 +121,21 @@ public function getApplicants(
                       AND `applicants`.`stageId` like :stageId 
                       GROUP BY `applicants`.`id`";
         $stmt .= $this->sortingQuery($sortingQuery);
-        $stmt .= " LIMIT :offsets, :numberPerPage;";
-        $offset = ($pageNumber - 1) * $this->numberPerPage;
         $query = $this->db->prepare($stmt);
         $query->setFetchMode(\PDO::FETCH_CLASS, BaseApplicantEntity::class);
         $query->bindValue(':name', $name);
         $query->bindValue(':stageId', $stageId);
         $query->bindValue(':cohortId', $cohortId);
         $query->bindValue(':cohortId2', $cohortId);
-        $query->bindValue(':offsets', $offset, \PDO::PARAM_INT);
-        $query->bindValue(':numberPerPage', $this->numberPerPage, \PDO::PARAM_INT);
         $query->execute();
-        $applicants = $query->fetchAll();
-        foreach ($applicants as $key => $applicant) {
+        $allApplicants = $query->fetchAll();
```

Comment:
no need to rename this from `$applicants`

Reply:
changed back